chart/compass/values.yaml

```diff
@@ -742,8 +742,8 @@ global:
     systemsAPIEndpoint: ""
     # systemsAPIFilterCriteria - criteria for fetching systems
     systemsAPIFilterCriteria: ""
-    # systemToTemplateMappings - how to map system properties to an existing application template
-    systemToTemplateMappings: '{}'
+    appTemplatesLabelFilter: "systemRole"
```

Comment:
Can we rename `appTemplatesLabelFilter` to `appTemplatesProductLabel`? Because the value of this property is practically the label value which we put to app templates, not an actual filter. It just so happens that we later filter based on that.